### PR TITLE
Make the code work with github 0.9.6 and 1.2.0

### PR DESCRIPTION
**Pull request by mscherer**
So we can upgrade the module without breakage and synchronisation

### CODE REVIEW

**Comment by mscherer** [comment]
@deepshikhaaa So we are not stuck with older release of github3.py

File: github/handle_github.py [hunk 1]
```diff
@@ -90,7 +90,13 @@ def check_issue(self, num):
...
                 self.error_string.append(error)
                 return False
-            for label in issue.labels:
+            # compatibility between 1.20 and 0.9.6
+            if type(issue.labels) == type([]):
+                l = issue.labels
+            else:
+                l = issue.labels()
+
+            for label in l:
```
Comment (deepshikhaaa) [inline github/handle_github.py]: If issue.labels is not list or not of any iterable type, it will not iterate. Here in newer version it is issue.labels().

Comment (mscherer) [reply github/handle_github.py]: issue.label on newer github3.py is a function that return a iterable and a list on the old version, so I am not sure if I should change something or not ?

Comment (deepshikhaaa) [reply github/handle_github.py]: Now I see your point. 
LGTM

**Pull request merged**